### ISSUE TITLE
App Compatibility for MacOS mojave 

### ISSUE THREAD

**Issue by Crozaniz** [modrinth/code#181]
### Describe the bug

So basically what i think would be great if modrinth compatibility with MacOS mojave was a reality, This isn't really a Bug/issue but more of a Suggestion. People who cant use Modrinth with MacOS mojave is really sad, im saying here that maybe make mojave the lowest you can get to gain access to the Modrinth App, and just leave High Sierra, Sierra, or maybe even yosemite, Without access to the Modrinth App. 

This is a pretty short suggestion but hopefully it works!

### Expected behavior

Nothing much but just opinions from people and even moderators to see if they want this in modrinth (mac users)

**Comment by Imzxhir** [comment]
I would like to say that the Modrinth Launcher is compatible with High Sierra, so I do not understand why you are asking for Mojave support when it should be already supported.

**Comment by Crozaniz** [comment]
Odd because when i tried launching modrinth with mojave it just completely tells me its not compatible, let me try again to see if it works

**Comment by Crozaniz** [comment]
Nope! it doesn't work

It says "Modrinth app cannot be opened because of a problem.
Check with the developer to make sure Modrinth App works with this version of macOS. You may need to reinstall the application. Be sure to install any available updates for the application and macOS.

Click Report to see more detailed information and send a report to Apple."

**Comment by Crozaniz** [comment]
So technically this is an issue now or its just my macOS but thats very low probability.

**Comment by Crozaniz** [comment]
> So technically this is an issue now or its just my mac but thats very low probability.

its basically just skipping mojave.

**Comment by Imzxhir** [comment]
What version of macOS are you on?

**Comment by bigmaxine** [comment]
Not on macOS Mojave but this also happens on my Mid-2014 Macbook Pro running Big Sur

**Comment by Crozaniz** [comment]
> What version of macOS are you on?

MacOS mojave 1.14.6

**Comment by Prospector** [comment]
Going to close, the version of macOS in question is so old that it doesn't seem worth fixing. I'd encourage you to use another launcher like Prism Launcher to be able to easily install Modrinth from a launcher.
